src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const employees = [...document.querySelectorAll('li')];
+
+function sortEmployees(arr) {
+  const salary = (item) => +item.slice(1).split(',').join('');
+
+  const sorted = arr.sort((a, b) => {
+    return salary(b.dataset.salary) - salary(a.dataset.salary);
+  });
+
+  list.append(...sorted);
+};
+
+function getEmployees(arr) {
+  return arr.map((item) => {
+    return {
+      name: item.innerText,
+      position: item.dataset.position,
+      salary: item.dataset.salary,
+      age: item.dataset.age,
+    };
+  });
```

Comment:
Destructuring make it more clear
```suggestion
  return arr.map(({innerText, dataset: {position, salary, age}}) => {
    return {
      name,
      position,
      salary,
      age,
    };
  });
```